src/log4stash/ElasticClient/WebElasticClient.cs

```diff
@@ -27,8 +27,8 @@ private IRestClient RestClient
         private readonly IResponseValidator _responseValidator;
         private readonly IExternalEventWriter _eventWriter;
 
-        public WebElasticClient(IServerDataCollection servers, int timeout)
-            : this(servers, timeout, false, false, new AuthenticationMethodChooser())
+        public WebElasticClient(IServerDataCollection servers, IExternalEventWriter eventWriter, int timeout)
+            : this(servers, timeout, false, false, new AuthenticationMethodChooser(eventWriter))

```

Comment:
Pay attention that we already have `_eventWriter` on this class - I think in any case it is best to change the arguments order and use only one eventWriter (also need to update the factory, I think just inline `new LogLogEventWriter()` there should be ok)